src/AccessibilityInsights.SharedUx/FileIssue/IssueReporterManager.cs

```diff
@@ -42,22 +46,28 @@ private IssueReporterManager()
         }
 
         // Unit testing constructor
-        internal IssueReporterManager(ConfigurationModel configs, IEnumerable<IIssueReporting> issueReportingOptions)
+        internal IssueReporterManager(ConfigurationModel appConfig, IEnumerable<IIssueReporting> issueReportingOptions)
         {
-            var serializedConfigsDict = configs.IssueReporterSerializedConfigs;
-            Dictionary<Guid, string> configsDictionary = !string.IsNullOrWhiteSpace(serializedConfigsDict) ?
-                JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigsDict)
-                : new Dictionary<Guid, string>();
+            if (appConfig == null)
+                throw new ArgumentNullException(nameof(appConfig));
+
+            _appConfig = appConfig;
+            _issueReportingOptions = issueReportingOptions ?? Enumerable.Empty<IIssueReporting>();
+        }
+
+        internal void RestorePersistedConfigurations()
```

Comment:
I changed this class to use 2-step initialization instead of doing everything in the ctor. Doesn't impact the overall effect, but it caused changes in the unit tests